apps/src/lib/levelbuilder/lesson-editor/ActivitiesEditor.story.jsx

```diff
@@ -10,23 +10,42 @@ import resourcesEditor, {
 import {Provider} from 'react-redux';
 import {
   sampleActivities,
+  sampleActivityForLessonWithoutLessonPlan,
   searchOptions
 } from '../../../../test/unit/lib/levelbuilder/lesson-editor/activitiesTestData';
 
-const createStore = () => {
+const createStoreLessonWithLessonPlan = () => {
```

Comment:
nit: we're still creating a store, right? If so, could this be changed to `createStoreWIthLessonPlan`?